### PR TITLE
Publish minutes for 2016-11-04 networking meeting

### PR DESCRIPTION
**Pull request by seabaylea**
Minutes from Friday's kick-off call for the networking sub-team.

@swift-server/stakeholders please review.

### CODE REVIEW

**Review by adellibovi — APPROVED** [review]

File: meetings/networking/2016-11-04/minutes.md [hunk 1]
```diff
@@ -1,17 +1,76 @@
...
+* Johannes Weiss - Apple, iCloud
+* Gregor Milos - Apple, iCloud
+* Scott Lessans - Full Stack/Robotics; nourish.ai
+* Ben Cohen - Swift standard lib @Apple
+
+### Minutes:
+
+_Q) Don’t we just need a layer on POSIX? Or just the intent of the POSIX layer? How are developers going to use the APIs in the most effective way? Do they really need the low ever APIs?_  
+* Devs really just want to use high lebel APIs that are focused on their use case.  
+* We we take the approach of building the high level APIs, and use that to inform us of what the lower level capabilities need to be?
```
Comment (adellibovi) [inline meetings/networking/2016-11-04/minutes.md]: Small typo: We take

Comment (baberthal) [reply meetings/networking/2016-11-04/minutes.md]: Another typo: `high lebel` vs `high level`

**Comment by baberthal** [comment]
@seabaylea - I created a PR on your fork with fixes to the typos I spotted.

**Comment by seabaylea** [comment]
Breaking the rules and self merging, based in the stakeholder reviews.

**Pull request merged**